dropbox/session.py

```diff
@@ -32,22 +32,33 @@
 # This is the default longest time we'll block on receiving data from the server
 DEFAULT_TIMEOUT = 100
 
-_TRUSTED_CERT_FILE = pkg_resources.resource_filename(__name__, 'trusted-certs.crt')
+try:
+    _TRUSTED_CERT_FILE = pkg_resources.resource_filename(__name__, 'trusted-certs.crt')
+except NotImplementedError:  # Package is used inside python archive
+    _TRUSTED_CERT_FILE = None
+
 
 # TODO(kelkabany): We probably only want to instantiate this once so that even
 # if multiple Dropbox objects are instantiated, they all share the same pool.
 class _SSLAdapter(HTTPAdapter):
+
+    def __init__(self, *args, **kwargs):
+        self._ca_certs = kwargs.pop("ca_certs", None) or _TRUSTED_CERT_FILE
+        if not self._ca_certs:
+            raise FileNotFoundError("CA certificate not found")
```

Comment:
This error is a bit misleading - it's not checking for the presence of a readable file (although maybe it should be?) - it's simply checking to see if `ca_certs` or `_TRUSTED_CERT_FILE` is set.